application/classes/Ushahidi/Validator/Tag/Update.php

```diff
@@ -54,7 +54,8 @@ protected function getRules()
 				['color'],
 			],
 			'icon' => [
-				['alpha_dash'],
+				// alphas, numbers, punctuation, and spaces
```

Comment:
I'll pull out the punctuation, none of the font awesome icons have numbers so I guess those can be removed until we hit an icon pack that has them.